gh-projects/assign-to-project/src/actions/assignProject.test.ts

```diff
@@ -0,0 +1,116 @@
+import * as octokitGraphql from '@octokit/graphql';
```

Comment:
Is this test run in CI? I only see `lint` and `build` steps there.

Reply:
The test is inside the build step I will rename it to test